docs/v3-ea-components/transport-types/http-transport.md

```diff
@@ -1,3 +1,135 @@
 # HTTP Transport
 
-_Guide coming soon_
+HttpTransport is used to fetch data from a Provider using HTTP(S) protocol. In order to use HttpTransport you need to provide two methods `prepareRequests` and `parseResponse` that will be used to build requests and parse responses respectively. 
+
+Example usage of HttpTransport
+```typescript  
+const transport = new HttpTransport<EndpointTypes>({  
+  prepareRequests: (params, config) => {  
+    return params.map((param) => {  
+      const symbol = param.symbol.toLowerCase()  
+      const url = `/price/${symbol}`  
+  
+      return {  
+        params: param  
+		request: {  
+          baseURL: config.API_ENDPOINT,  
+          url,  
+        },  
+      }  
+    })  
+  },  
+  parseResponse: (params, res, config) => {  
+    return res.data.map((result) => {  
+      return {  
+        params: { symbol: result.symbol},  
+        response: {  
+          data: {  
+            result: result.price,  
+          },  
+          result: result.price,  
+        },  
+      }  
+    })  
+  },  
+})
+```
+### Building requests
```

Comment:
Another nitpick.

I can't do a suggested change because the formatting gets messed up, but we should have one open line between L36-L37 and L37-L38.